charts/datree-admission-webhook/templates/wait-server-ready-post-install.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: {{template "datree.namespace" .}}
   labels: {{ include "datree.labels" . | nindent 4 }}
   annotations:
-    "helm.sh/hook": post-install
+    "helm.sh/hook": post-install, post-upgrade
```

Comment:
I did another test, it also works with *only* this change (and the image update)🤔😅
@niv1612 

We need to figure what's going on here